frontend/public/components/_quota.scss

```
@@ -0,0 +1,7 @@
+.co-m-resource-quota-icon {
```

We should drop the `-m-`.

```suggestion
.co-resource-quota-icon {
```

---

fixed